testsuite/shared/src/main/java/org/jboss/as/test/integration/common/jms/JMSOperations.java

```diff
@@ -76,4 +80,7 @@
 
     void removeSystemProperties();
 
+    void addSocketBinding(String bindingName, ModelNode... clientMappings);
```

Comment:
I'd prefer that such methods are not added to the JMSOperations interface.
adding/removing a socket-binding is not an operation that is related to the messaging subsystems.
The op is simple enough that it can be coded directly or provided by a helper class.